src/guake/gconfhandler.py

```diff
@@ -364,7 +364,6 @@ def reload_globals(self, client, connection_id, entry, data):
         method. If more global hotkeys should be added, just connect
         the gconf key to the watch system and add.
         """
-        self.guake.hotkeys.unbind_all()
```

Comment:
It will be a problem if their is gonna be more places in Guake where you can change your keybindings, because they will all have too have code that unbind the previous key. But we could bundle a modifies python binding that has the unbind_all function exposed in python. If you interested I would gladly try to implement.